charts/opentelemetry-collector/values.yaml

```diff
@@ -1,6 +1,10 @@
+
 # Default values for opentelemetry-collector.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+global:
+  # global.imageRegistry -- Global Docker image registry
+  imageRegistry: ""
```

Comment:
We only have 1 image in this helm chart so I'm not sure what this is providing us.  If you're trying to us this chart as a subchart and the `"additionalProperties": false,` schema is preventing the failure we can update the schema file.  We already have some other common `global` values in there.

Reply:
Thanks for the suggestion @TylerHelmuth . Indeed I'm trying to use this chart as a subchart and I found the `additionalProperties": false` is not blocking me at all. I just dropped the irrelevant `global.imageRegistry`